### PR TITLE
chore: use official docker image and cleanup

### PR DESCRIPTION
**Pull request by vigneshshanmugam**
+ Part of #61 
+ Cleans up the customer docker related stuff we have for synthetics prior to 7.12 versions. 
+ Uses the official beats heartbeat image for all kinds of testing and also updated the heartbeat monitor config with the latest syntax changes.

### CODE REVIEW

**Comment by vigneshshanmugam** [comment]
@kuisathaverat I am planning to comment out the section in the Jenkins pipeline where we publish to the experimental namespace. Do you have any opposition?

**Comment by kuisathaverat** [comment]
I think you can remove it, we are releasing the official Heartbeat Docker image.

**Comment by kuisathaverat** [comment]
we have to rework on https://github.com/elastic/synthetics/pull/193 to test with the heartbeat Docker image

File: __tests__/e2e/docker-compose.yml [hunk 1]
```diff
@@ -44,13 +44,13 @@ services:
     networks:
       - elastic
   synthetic:
-    image: docker.elastic.co/experimental/synthetics:${SYNTHETICS_VERSION}-synthetics
```
Comment (kuisathaverat) [inline __tests__/e2e/docker-compose.yml:47]: this will not test the current commit, I am changing it on https://github.com/elastic/synthetics/pull/193

Comment (kuisathaverat) [reply __tests__/e2e/docker-compose.yml:47]: I just see that the code is mounted as volume.

**Review by kuisathaverat — APPROVED** [review]

File: __tests__/e2e/scripts/setup.sh [hunk 2]
```diff
@@ -33,4 +33,4 @@ echo "" # newline
 echo "${bold}Starting elasticsearch , kibana and synthetics docker${normal}"
 echo "" # newline
 
-STACK_VERSION=8.0.0-SNAPSHOT SYNTHETICS_VERSION=8.0.0 docker-compose --file docker-compose.yml up --remove-orphans > ${TMP_DIR}/docker-logs.log 2>&1 &
+STACK_VERSION=8.0.0-SNAPSHOT docker compose --file docker-compose.yml up --remove-orphans > ${TMP_DIR}/docker-logs.log 2>&1 &
```
Comment (vigneshshanmugam) [inline __tests__/e2e/scripts/setup.sh:36]:
```suggestion
STACK_VERSION=8.0.0-SNAPSHOT docker-compose --file docker-compose.yml up --remove-orphans > ${TMP_DIR}/docker-logs.log 2>&1 &
```

**Pull request merged**